### PR TITLE
alacritty: link extra manpage and terminfo entries

### PR DESCRIPTION
**Pull request by carlocab**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [ ] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [ ] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [ ] `brew audit --new-cask <cask>` worked successfully.
- [ ] `brew install --cask <cask>` worked successfully.
- [ ] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/alacritty.rb [hunk 1]
```diff
@@ -20,7 +20,12 @@
          target: "#{HOMEBREW_PREFIX}/etc/bash_completion.d/alacritty"
   binary "#{appdir}/Alacritty.app/Contents/Resources/completions/alacritty.fish",
          target: "#{HOMEBREW_PREFIX}/share/fish/vendor_completions.d/alacritty.fish"
+  binary "#{appdir}/Alacritty.app/Contents/Resources/61/alacritty",
+         target: "#{ENV["HOME"]}/.terminfo/61/alacritty"
```
Comment (imomaliev) [inline Casks/alacritty.rb:24]: Thanks for the additions. Some people use [`TERMINFO`](https://invisible-island.net/ncurses/man/ncurses.3x.html#h3-TERMINFO) and `TERMINFO_DIRS`. Could we add support for them?

Comment (carlocab) [reply Casks/alacritty.rb:24]: It's not clear to me how to do so, but PRs to do that are welcome.

Comment (imomaliev) [reply Casks/alacritty.rb:24]: I will try look into this tomorrow

Comment (imomaliev) [reply Casks/alacritty.rb:24]: @carlocab https://github.com/Homebrew/homebrew-cask/pull/119610 here is POC

**Pull request merged**